app/models/draft-registration.ts

```diff
@@ -23,6 +35,21 @@ export default class DraftRegistrationModel extends OsfModel {
 
     @belongsTo('registration-schema', { inverse: null })
     registrationSchema!: DS.PromiseObject<RegistrationSchemaModel> & RegistrationSchemaModel;
+
+    @hasMany('contributor', { inverse: 'node' })
+    contributors!: DS.PromiseManyArray<ContributorModel>;
+
+    @hasMany('contributor', { inverse: null })
+    bibliographicContributors!: DS.PromiseManyArray<ContributorModel>;
+
+    @hasMany('institution', { inverse: 'nodes' })
+    affiliatedInstitutions!: DS.PromiseManyArray<InstitutionModel> | InstitutionModel[];
```

Comment:
```suggestion
    affiliatedInstitutions!: DS.PromiseManyArray<InstitutionModel>;
```
institutions are the only has-many relation in the app that's typed with `| T[]` -- i don't know why that is, but i feel like we should make it consistent with the other relations unless/until that causes a problem